### ISSUE TITLE
Remark plugin cache error: ENOENT

### ISSUE THREAD

**Issue by lucasecdb** [gatsbyjs/gatsby#18043]
## Description

I'm trying to run `gatsby develop`, but the command spits out several "info: create file" inside the `.cache` directory of my app. It's not preventing me from running the app, but it's really annoying and makes the build really slow.

### Steps to reproduce

I'm running this into [this repository](https://github.com/OpenDevUFCG/Tamburetei), the app is inside the `_app` folder.

### Expected result

I didn't expect these errors to occur, since I've run this project in another machine and it worked fine. I also have another gatsby project that runs fine in this computer.

### Actual result

I've recorded a [asciinema](https://asciinema.org/a/272043) video to help you see the problem.

### Environment

```
  System:
    OS: macOS 10.14.6
    CPU: (4) x64 Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz
    Shell: 5.3 - /bin/zsh
  Binaries:
    Node: 12.10.0 - /usr/local/bin/node
    Yarn: 1.19.0 - /usr/local/bin/yarn
    npm: 6.11.3 - /usr/local/bin/npm
  Languages:
    Python: 2.7.10 - /usr/bin/python
  Browsers:
    Chrome: 77.0.3865.90
    Firefox: 69.0
    Safari: 13.0.1
  npmPackages:
    gatsby: ^2.15.28 => 2.15.28 
    gatsby-image: ^2.2.23 => 2.2.23 
    gatsby-plugin-google-fonts: ^1.0.1 => 1.0.1 
    gatsby-plugin-manifest: ^2.2.20 => 2.2.20 
    gatsby-plugin-react-helmet: ^3.1.10 => 3.1.10 
    gatsby-plugin-sass: ^2.1.17 => 2.1.17 
    gatsby-plugin-sharp: ^2.2.27 => 2.2.27 
    gatsby-source-filesystem: ^2.1.28 => 2.1.28 
    gatsby-transformer-remark: ^2.6.26 => 2.6.26 
    gatsby-transformer-sharp: ^2.2.19 => 2.2.19 
```

**Comment by lucasecdb** [comment]
I also want to point out that removing the `gatsby-transformer-remark` from my `gatsby-config.js` "solves" the problem. But I'm not able to use my markdown files without it :-(

**Comment by thayannevls** [comment]
I'm having the same problem :(

**Comment by lucasecdb** [comment]
fixed in OpenDevUFCG/Tamburetei#282
